boards/arm/nrf5340_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,260 @@
+.. _nrf5340_dk_nrf5340:
+
+nRF5340-DK-NRF5340
+##################
+
+Overview
+********
+
+The nRF5340 DK NRF5340 is a single-board development kit for evaluation
```

Comment:
Replace with nRF5340 DK, remove the extra NRF5340. The DK is called nRF5340 DK.

Reply:
Alright, will do.